src/Http/Controllers/VoyagerBreadController.php

```diff
@@ -156,7 +156,7 @@ public function update(Request $request, $id)
         $this->insertUpdateData($request, $slug, $dataType->editRows, $data);
 
         return redirect()
-            ->route("voyager.{$dataType->slug}.index")
+            ->route("voyager.{$dataType->slug}.edit", ['id' => $id])
```

Comment:
This break the tests. On the other hand, this change has been discussed earlier in #720.